lib/container_orchestrator.rb

```diff
@@ -0,0 +1,28 @@
+class ContainerOrchestrator
+  TOKEN_FILE = "/run/secrets/kubernetes.io/serviceaccount/token".freeze
+
+  def scale(deployment_config_name, replicas)
+    connection.patch_deployment_config(deployment_config_name, { :spec => { :replicas => replicas } }, ENV["MY_POD_NAMESPACE"])
+  end
+
+  private
+
+  def connection
+    require 'kubeclient'
+
+    @connection ||=
+      Kubeclient::Client.new(
+        manager_uri,
+        :auth_options => { :bearer_token_file => TOKEN_FILE },
+        :ssl_options  => { :verify_ssl => OpenSSL::SSL::VERIFY_NONE }
```

Comment:
friendly reminder: will need something better than VERIFY_NONE eventually :)

This is to be used when MIQ runs inside the cluster, right?
https://kubernetes.io/docs/tasks/tls/managing-tls-in-a-cluster/#trusting-tls-in-a-cluster
sounds like we can use /var/run/secrets/kubernetes.io/serviceaccount/ca.crt